### PR TITLE
Remember drawer expanded state across sessions

### PR DESCRIPTION
**Pull request by spadgett**
/assign @jwforres 
@benjaminapetersen

### CODE REVIEW

**Comment by spadgett** [comment]
Fixing pet peeves :)

**Comment by spadgett** [comment]
This is consistent with how we remember the nav expanded state.

https://github.com/openshift/origin-web-console/blob/master/app/scripts/directives/nav.js#L203-L219

File: app/scripts/directives/notifications/notificationDrawerWrapper.js [hunk 1]
```diff
@@ -246,7 +249,7 @@
       angular.extend(drawer, {
         drawerHidden: true,
         allowExpand: true,
```
Comment (benjaminapetersen) [inline app/scripts/directives/notifications/notificationDrawerWrapper.js]: Ah so you would prefer it would remember this state

Comment (spadgett) [reply app/scripts/directives/notifications/notificationDrawerWrapper.js]: The idea is if you prefer it open to see descriptions, you don't have to click the little expand button every time. And we tend to remember other collapsed / expanded states like overview list rows and nav.

**Pull request merged**